bot/src/whitelist.ts

```diff
@@ -1,20 +1,41 @@
 import { getCollateralConfigByType } from 'auctions-core/src/constants/COLLATERALS';
 import { WHITELISTED_COLLATERALS } from './variables';
 
-const validateWhitelist = function (whitelist: string[]) {
-    whitelist.forEach(collateralType => {
-        getCollateralConfigByType(collateralType);
+const validateWhitelist = function (whitelist: string[]): Promise<void> {
+    return new Promise((resolve, reject) => {
+        const errors: string[] = [];
```

Comment:
Those are not `errors`, but `notFoundCollateralTypes` or `unsupportedCollateralTypes`